extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -154,6 +156,39 @@ export const LIFECYCLE_STAGES = {
   resumeCallback: '27',
   visIniLoad: '29',
   upgradeDelay: '30',
+  // TODO(warrengm): This should replace xDomIframeLoaded once delayed fetch
+  // is fully deprecated. A new lifecycle stage, crossDomainIframeLoaded, was
+  // introduced since xDomIframeLoaded is handled in AmpAdXOriginIframeHandler
+  // outside A4A.
+  crossDomainIframeLoaded: '31',
+};
+
+/**
+ * Name of A4A lifecycle triggers.
+ * @enum {string}
+ */
+export const AnalyticsTrigger = {
+  AD_REQUEST_START: 'ad-request-start',
+  AD_RESPONSE_END: 'ad-response-end',
+  AD_RENDER_START: 'ad-render-start',
+  AD_RENDER_END: 'ad-render-end',
+  AD_IFRAME_LOADED: 'ad-iframe-loaded',
+};
+
+/**
+ * Maps the names of lifecycle events to analytics triggers.
+ * @const {!Object<string, !AnalyticsTrigger>}
```

Comment:
Mark as @private and add _ to variable name

Reply:
This variable is unexported so I don't think a visibility modifier is needed. Other module locals (NO_CONTENT_RESPONSE, etc.) lack visibility modifiers. I can change it if it's required due to some compiler settings or due to a style guide.